org.eclipse.winery.repository.ui/src/app/wineryTargetAllocation/policySelection/policySelection.component.ts

```diff
@@ -0,0 +1,134 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0, or the Apache Software License 2.0
+ * which is available at https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR Apache-2.0
+ *******************************************************************************/
+
+import { Component, OnInit, ViewChild } from '@angular/core';
+import { WineryTableColumn } from '../../wineryTableModule/wineryTable.component';
+import { SelectComponent, SelectItem } from 'ng2-select';
+import { WineryNotificationService } from '../../wineryNotificationModule/wineryNotification.service';
+import { Policy, TargetAllocationService } from '../targetAllocation.service';
+
+@Component({
+    selector: 'winery-target-allocation-policy-selection-component',
+    templateUrl: 'policySelection.component.html'
+})
+
+export class PolicySelectionComponent implements OnInit {
+
+    @ViewChild('property') propertySelect: SelectComponent;
+
+    // policy, property, operator selection
+    policiesForSelect: Array<any> = [];
+    propertiesForSelect: Array<any> = [];
```

Comment:
`{ id: key, text: key }` is used here: you could reuse the `SelectItem` interface here